go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -87,7 +87,7 @@ add rule inet ovn-kubernetes ovn-kube-local-gw-masq jump ovn-kube-udn-masq
 
 const baseLGWNFTablesRules = `
 add rule inet ovn-kubernetes ovn-kube-local-gw-masq ip saddr 169.254.169.1 masquerade
-add chain inet ovn-kubernetes ovn-kube-local-gw-masq { type nat hook postrouting priority 100 ; comment "OVN local gateway masquerade" ; }
+add chain inet ovn-kubernetes ovn-kube-local-gw-masq { type nat hook postrouting priority 101 ; comment "OVN local gateway masquerade" ; }
```

Comment:
i think the knftables code will already handle flushing and adding new rules on upgrade, but can you double check?

Reply:
https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5140#issuecomment-3114142964